delete-provenance-artifacts.sh

```diff
@@ -4,18 +4,16 @@
 #cp kubeprovenance ./artifacts/simple-image/kube-provenance-apiserver
 #docker build -t kube-provenance-apiserver:latest ./artifacts/simple-image
 
-kubectl delete ns provenance
-kubectl delete -f artifacts/example/auth-delegator.yaml -n kube-system
-kubectl delete -f artifacts/example/auth-reader.yaml -n kube-system
-kubectl delete -f artifacts/example/apiservice.yaml
-kubectl delete -f artifacts/example/grant-cluster-admin.yaml
+#cluster/kubectl.sh should be in your PATH env var
+kubectl.sh delete ns provenance
+kubectl.sh delete -f artifacts/example/auth-delegator.yaml -n kube-system
+kubectl.sh delete -f artifacts/example/auth-reader.yaml -n kube-system
+kubectl.sh delete -f artifacts/example/apiservice.yaml
+kubectl.sh delete -f artifacts/example/grant-cluster-admin.yaml
+#FOR MINIKUBE UNCOMMENT THESE:
```

Comment:
What is the difference between lines 8-12 and 14-18?

Reply:
the .sh, kubectl.sh instead of kubectl .. running it locally as was said in that tutorial you linked me awhile for running kubernetes on gce a while ago, I have to run this shell file in cluster/kubectl.sh..

Reply:
ah okay.